### PR TITLE
Changed scaling of A3C loss function

### PR DESCRIPTION
**Pull request by peastman**
I changed `reduce_sum()` to `reduce_mean()`.  Previously, the loss was proportional to the rollout length, which meant you had to adjust the learning rate based on the rollout length.  This change makes it a bit more convenient to work with.

### CODE REVIEW

**Comment by rbharath** [comment]
LGTM
